catroid/src/main/java/org/catrobat/catroid/content/bricks/LegoEv3MotorMoveBrick.java

```diff
@@ -151,8 +147,11 @@ public void onItemSelected(AdapterView<?> arg0, View arg1, int position, long ar
 			public void onNothingSelected(AdapterView<?> arg0) {
 			}
 		});
-
+		if (motorEnum == null) {
+			readResolve();
+		}
 		motorSpinner.setSelection(motorEnum.ordinal());
+		motorSpinner.setGravity(Gravity.CENTER);
```

Comment:
can you do this in the layout XML files?
I don't know if the XML files will change due to refactoring, but definitely not in the first iteration.
I see potential problems if part of the appearance is defined in the XML and part of it in code.